packages/ios-client/Sources/FishjamClient/CommandsQueue.swift

```diff
@@ -1,5 +1,13 @@
 import Promises
 
+class CommandQueueError: Error {
+    let message: String
+
+    init(message: String) {
```

Comment:
**suggestion:** 
To keep it shorter i suggest:
```suggestion
    init(_ message: String) {
```

Reply:
I'm not too sure about that one. It will be shorter, true but it may not read that good that it's supposed to be a message